cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -48,23 +50,29 @@ func NewCmd() *cobra.Command {
 
 			var err error
 			o := scorecard.Scorecard{
-				ServiceAccount: serviceAccount,
-				Namespace:      namespace,
-				BundlePath:     bundle,
-				SkipCleanup:    skipCleanup,
-				WaitTime:       waitTime,
+				SkipCleanup: skipCleanup,
 			}
 
 			if bundle == "" {
 				return fmt.Errorf("bundle flag required")
 			}
 
-			o.Client, err = scorecard.GetKubeClient(kubeconfig)
+			runner := scorecard.PodTestRunner{
+				ServiceAccount: serviceAccount,
+				Namespace:      namespace,
+				BundlePath:     bundle,
+			}
+
+			runner.Client, err = scorecard.GetKubeClient(kubeconfig)
 			if err != nil {
 				return fmt.Errorf("could not get kubernetes client: %w", err)
 			}
 
-			o.Config, err = scorecard.LoadConfig(config)
+			configPath := filepath.Join(bundle, "tests", "scorecard", "config.yaml")
+			if config != "" {
+				configPath = config
+			}
+			o.Config, err = scorecard.LoadConfig(configPath)
```

Comment:
Looks good! For follow-on, we need to consider how we'll handle this if/when the bundle is an image and not a directory.